server/src/main/java/org/opensearch/index/engine/OpenSearchReaderManager.java

```diff
@@ -38,13 +38,8 @@
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.apache.lucene.index.DirectoryReader;
-import org.apache.lucene.index.LeafReader;
-import org.apache.lucene.index.LeafReaderContext;
-import org.apache.lucene.index.SegmentInfos;
-import org.apache.lucene.index.StandardDirectoryReader;
+import org.apache.lucene.index.*;
```

Comment:
we should leave the specific imports as is here.  My Intellij sometimes does this by default and had to change in Preferences > code style > java and mark use single class import and change `Class count to use import with '*' to a high number.